src/components/app-header/app-header.module.css

```
@@ -2,26 +2,26 @@
   background-color: #1C1C21;
 }
 
-.content {
+.header__wrapper {
   display: flex;
   justify-content: space-between;
   align-items: center;
   width: 1280px;
   margin: auto;
 }
 
-.list {
+.menu__list {
   display: flex;
   list-style: none;
   margin: 0;
   padding: 0;
 }
 
-.listItem:nth-child(n + 2) {
+.menu__listItem:nth-child(n + 2) {
```

Можно лучше:
если имя элемента составное, оно разделяется черточкой
```suggestion
.menu__list-item:nth-child(n + 2) {
```